docs/pages/access-controls/access-graph.mdx

```diff
@@ -23,174 +23,171 @@ under the Permission Management section.
 
-8. Resources
+The number on the right hand side shows the number of resources that a resource group contains.
+
+6. Resources
 
 ![Resource Node](../../img/access-graph/resource-node.png)
 
 Resources are the things that users can or cannot access. They can be
 servers, databases, or Kubernetes clusters.
 
-## Searching
-
-To search for a graph node, either click the search button in the sidebar on the left,
-or press the `s` key.
+## Graph paths
 
-![Search](../../img/access-graph/search.png)
+Teleport Access Graph shows the relationships between users, roles, and
+resources. It does this by showing paths between nodes. Paths are the
+relationships between nodes. Paths always connect nodes in the following order:
 
-You can then search through all node types. If you select a result using your keyboard
-arrow keys, you can press either `Tab` to add the node to the graph, or `Enter` to replace
-the graph with just the individual node.
+1. Users
+1. User Groups
+1. Actions
+1. Resource Groups
+1. Resources
 
-## Changing what is visible
+Paths can be divided into two categories:
 
-By default, Teleport Access Graph shows all types of nodes and paths. You can see what
-node types are visible by looking at the sidebar on the left.
+1. Allow paths
 
-![Sidebar](../../img/access-graph/sidebar.png)
+![Allow Path](../../img/access-graph/allow-path.png)
 
-### Showing/removing individual graph nodes
+Allow paths connect identities to resources. They show what an identity can access
+and what actions they can perform.
 
-To show or remove an individual graph node, open up the node type in the sidebar and
-click on the node you want to show or remove.
+2. Deny paths
 
-![Show/Remove Node](../../img/access-graph/search-item-click.png)
+![Deny Path](../../img/access-graph/deny-path.png)
 
-Clicking on the node name will toggle the visibility of that node.
+Deny paths connect identities to resources. They show what a identity cannot access
+and what actions they cannot perform. Deny paths take precedence over allow
+paths.
 
-![Select only one node](../../img/access-graph/search-item-checkbox-click.png)
+## How to use it
 
-Clicking on the checkbox will select only that node.
+Teleport Access Graph can help you to answer questions like:
 
-### Adding/removing graph node types
+- Who can access a specific resource?
 
-To add a graph node type that isn't in the view, select it from the list of available
-node types.
+![Show Access Path Resource](../../img/access-graph/show-access-path-resource.gif)
 
-![Add node type](../../img/access-graph/add-node-type.png)
+- What resources can a specific user access?
 
-To remove the graph node type, expand the node type in the sidebar and click on the
-remove button.
+![Show Access Path](../../img/access-graph/show-access-path.gif)
 
-![Remove node type](../../img/access-graph/remove-node-type.png)
+## Navigation
 
-### Expanding/collapsing graph nodes
+![Navigation](../../img/access-graph/sidebar.png)
```

Comment:
Would just make a cleaner version of this graphic

Reply:
Updated https://github.com/gravitational/teleport/pull/41307/commits/bc473db2e79bccd6e8166219676bf6ed5aca523e